Sources/Brave/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2120,6 +2120,17 @@ public class BrowserViewController: UIViewController {
 
   func makeShareActivities(for url: URL, tab: Tab?, sourceView: UIView?, sourceRect: CGRect, arrowDirection: UIPopoverArrowDirection) -> [UIActivity] {
     var activities = [UIActivity]()
+    
+    if !url.isLocal, !InternalURL.isValid(url: url), !url.isReaderModeURL {
+      let copyCleanURLActivity = CopyCleanURLActivity() { 
```

Comment:
Lets use `BasicMenuActivity` instead of creating a new class :) I meant to delete all the ones we have that are empty callback mules but decided against it in my PR that introduced it

Reply:
Removal and changes at https://github.com/brave/brave-ios/pull/8626/commits/d8eefe2239d6804e8d041498ef23d10d33115c4a
Please check and let me know